src/sbt-test/sbt-scalafix/wrapper/test

```diff
@@ -0,0 +1,14 @@
+# check that missing rewrites are detected, with or without a warm compilation cache
+> set scalafixOnCompile := false
+-> lintAll
```

Comment:
on master
```
[info] [info] Non-compiled module 'compiler-bridge_2.12' for Scala 2.12.12. Compiling...
[info] [info] Running scalafix on 2 Scala sources
[error] /tmp/sbt_4021633e/wrapper/src/main/scala/example/Example.scala:3: error: not found: object imported
[error] import imported.Imported
[error]        ^
[info] [info]   Compilation completed in 14.904s.
[info] [warn] /tmp/sbt_4021633e/wrapper/src/main/scala/example/Example.scala:3:17: Unused import
[info] [warn] import imported.Imported
[info] [warn]                 ^
[info] [warn] one warning found
[info] [info] Done compiling.
[info] [success] Total time: 20 s, completed Jul 15, 2020, 11:52:59 AM
[error] x sbt-scalafix / wrapper 
[error]  Cause of test exception: {line 3}  Command succeeded but failure was expected
```

Reply:
I investigated where the concurrent `Non-compiled module 'compiler-bridge_2.12' for Scala 2.12.12. Compiling...`/`Done compiling` was coming from as it looked like it could race with the bytecode consumption in `scalafix`. It turns out to be from `Test / scalafix`, so I added caab820 to strengthen the test.

Without the fix, the test logs now shows:
```
[info] [info] Running scalafix on 2 Scala sources
[error] /tmp/sbt_93ed664/wrapper/src/main/scala/example/Example.scala:3: error: not found: object imported
[error] import imported.Imported
[error]        ^
[info] [success] Total time: 5 s, completed Jul 15, 2020, 5:29:15 PM
[error] x sbt-scalafix / wrapper 
[error]  Cause of test exception: {line 3}  Command succeeded but failure was expected

```